Tests/DeviceDetectorTest.php

```diff
@@ -638,16 +638,16 @@ public function getTypeMethodFixtures(): array
     /**
      * @dataProvider getTypeMethodFixtures
      */
-    public function testTypeMethods(string $ua, array $checkTypes): void
+    public function testTypeMethods(string $user_agent, array $check): void
```

Comment:
```
public function testTypeMethods(string $userAgent, array $deviceTypes): void
```
or 
```
public function testTypeMethods(string $useragent, array $deviceTypes): void
```
`$check` - just $check as if we are expecting a bool value

Reply:
clearly nobody understands that it must match the data in the yaml files or it is a PHP incompatibility
So I will add an array map to rename the keys 😁

Reply:
We have PHPCS rules syntax in the project, they must be followed even in tests. We do not intend to change or weaken the policy for the tests, as well as the keys in the tests (since the fixture files are used in other projects for other programming languages).

The main developers are used to the current PHPCS rules, I even got used to it myself after ~7 years of adding tests.

Reply:
Yup I totally understand and would have said the same

And we can not change the data as others may use it and it would be a breaking change

Let's go to adapt the data to the policy on the fly

Reply:
I pushed https://github.com/matomo-org/device-detector/pull/7727/commits/b45667a7376cdf0789a1b90cf454137f4bf9ffc1 but I still have some work to do